src/WebJobs.Script.WebHost/Management/LinuxSpecialization/PackageDownloadHandler.cs

```diff
@@ -133,6 +135,8 @@ private void AriaDownload(string directory, string fileName, Uri zipUri, bool is
                 _logger.LogError(msg);
                 throw new InvalidOperationException(msg);
             }
+            _logger.LogInformation($"Executed: {Aria2CExecutable} --allow-overwrite -x12 -d {directory} -o {fileName} '{Sanitizer.Sanitize(zipUri?.AbsoluteUri)}");
```

Comment:
Here and below, don't copy this line, since things can get out of sync in the future. Instead capture the command line in a local and use in both places, running the entire log line through sanitizer.

Reply:
Will do it.